miniapp/src/main/AndroidManifest.xml

```diff
@@ -15,10 +15,15 @@
 
         <!-- Change init order for sdk-utils so it's initialized first -->
         <provider
-          android:name="com.rakuten.tech.mobile.sdkutils.SdkUtilsInitProvider"
-          android:authorities="${applicationId}.SdkUtilsInitProvider"
-          android:exported="false"
-          android:initOrder="100"
-          tools:replace="android:initOrder" />
+            android:name="com.rakuten.tech.mobile.sdkutils.SdkUtilsInitProvider"
+            android:authorities="${applicationId}.SdkUtilsInitProvider"
+            android:exported="false"
+            android:initOrder="100"
+            tools:replace="android:initOrder" />
+
+        <meta-data  android:name="analytics.accountId"
```

Comment:
Hmm.. is it possible to define it as `com.rakuten.tech.mobile.miniapp.analytics.accountId` ? 🤔

Reply:
I think that dummy value is fine 😄.